### PR TITLE
Information about moderation on Django Hero form.

### PR DESCRIPTION
**Pull request by asendecka**
references #333

### CODE REVIEW

File: fundraising/forms.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ class DjangoHeroForm(forms.ModelForm):
     )
     is_visible = forms.BooleanField(
         required=False,
-        label='Yes, display my name, URL, and logo on this site.',
+        label='Yes, display my name, URL, and logo on this site. (It\'ll be displayed shortly after we verify it.)',
```
Comment (jezdez) [inline fundraising/forms.py]: Can you turn that into a double quote to prevent the backslash :neckbeard:

Comment (timgraham) [inline fundraising/forms.py]: Also multi-line the string as in the other labels since it exceeds 79 chars now

Comment (jezdez) [inline fundraising/forms.py]: :clap:

**Comment by timgraham** [comment]
:100: if you squash your commits.

**Pull request merged**